codegen/core/src/main/scala/com/lightbend/akkasls/codegen/ModelBuilder.scala

```diff
@@ -411,9 +412,16 @@ object ModelBuilder {
 
     Option(rawEntity.getName).filter(_.nonEmpty).map { name =>
       EventSourcedEntity(
-        FullyQualifiedName(name, protoReference),
+        FullyQualifiedName(name, name, protoReference, Some(fqnExtractor.fileDescriptorObject(descriptor.getFile))),
         rawEntity.getEntityType,
-        State(FullyQualifiedName(rawEntity.getState, protoReference)),
+        // FIXME this assumes the state is defined in the same proto file as the
+        // entity, which I don't think is necessarily true.
```

Comment:
At least that is how all our samples does it, I think that if we want to support that the state message is in some other package that will likely need changes elsewhere as well.

Reply:
yeah I copied this comment :-)